.github/workflows/installers-conda.yml

```
@@ -262,33 +283,48 @@ jobs:
         run: |
           $PKG_PATH -b
 
-          root_prefix=$(compgen -G $HOME/.local/spyder-*)
+          base_prefix=$(compgen -G $HOME/.local/spyder-*)
+          spy_rt=$base_prefix/envs/spyder-runtime
+          menu=$spy_rt/Menu/spyder-menu.json
+          mode=user
+          shortcut=$(${base_prefix}/bin/python - <<EOF
+          from menuinst.api import _load
+          menu, menu_items = _load("$menu", target_prefix="$spy_rt", base_prefix="$base_prefix", _mode="$mode")
+          print(menu_items[0]._paths()[0])
+          EOF
+          )
 
           echo "Install info:"
-          echo "Contents of ${root_prefix}:"
-          ls -al $root_prefix
-          echo -e "\nContents of ${root_prefix}/uninstall-spyder.sh:"
-          cat ${root_prefix}/uninstall-spyder.sh
-
-          shortcut_path=/share/applications/spyder_spyder.desktop
-          [[ -e ${root_prefix}/.nonadmin ]] && shortcut_path=${HOME}/.local${shortcut_path} || shortcut_path=/usr${shortcut_path}
-
-          if [[ -e $shortcut_path ]]; then
-              echo -e "\nContents of ${shortcut_path}:"
-              cat $shortcut_path
+          echo "Contents of ${base_prefix}:"
+          ls -al $base_prefix
+          echo -e "\nContents of ${base_prefix}/uninstall-spyder.sh:"
```

```suggestion

          if [[ -e ${base_prefix}/uninstall-spyder.sh ]]; then
              echo -e "\nContents of ${base_prefix}/uninstall-spyder.sh:"
              cat ${base_prefix}/uninstall-spyder.sh
          else
              echo "Uninstall script not available!"
              exit 1
          fi
```

Check for the existence of the uninstall script (please correct my bash code if it's not right by editing my suggestion before applying it).